cmds/core/pidof/pidof_linux.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
I think you can make this work on many kernels.
Plan 9, for example, has /proc
One way to do this: 
make pidof_<kernel>.go
have a function return a []struct{
name string
pid string
}
and then have pidof.go range over this slice and print the matching ones. 

If you can write this command in this manner, I will contribute pidof_plan9.go

Reply:
good point, updated.